### ISSUE TITLE
Error on fitting non-toa models when `local=T`

### ISSUE THREAD

**Issue by JosephReps** [b-steve/acre#45]
Was working on the estimated locations, when I realized there is currently a bug when trying to fit models using `local=T`.

Reprex:

```
ss.acre <- read.acre(ss$capt, ss$traps, control.mask = ss$control.mask)
ss.fit <- fit.acre(ss.acre, ss.opts = ss$ss.opts, local = T)
```

Results in:

```
Problem with these data entries:
toa_ssq 
     45 
Error in TMB::MakeADFun(data = data, parameters = parameters, map = map,  : 
  Only numeric matrices, vectors, arrays, factors, lists or length-1-characters can be interfaced
Called from: TMB::MakeADFun(data = data, parameters = parameters, map = map, 
    slient = (!tracing), DLL = "acre")
```

Note fitting toa models with `local=T` works fine.

Think I know where the problem is happening, will try to have fixed by time I land (airplane wifi sketchy).

**Comment by b-steve** [comment]
@JosephReps, I just a fixed a bug that prevented models from fitting with `local = TRUE` when there was not TOA data (see 57db2fec64cd01e553b3076cb8a69d51d879450a). Was this the same problem?

**Comment by JosephReps** [comment]
Not 100% sure, however if the code above runs now, then I'm assuming so. Might be worth trying to fit a model with each of the additional infotypes if you wanted to be sure?

**Comment by b-steve** [comment]
Yup, works fine. I figured out the problem and it won't affect other information types.
